files/apt/sources.list.amd64

```diff
@@ -3,13 +3,13 @@
 
 deb [arch=amd64] http://deb.debian.org/debian/ bullseye main contrib non-free
```

Comment:
Since we have `debian-archive.trafficmanager.net` providing the bullseye packages, I don't think we need `deb.debian.org` as well, right?

Reply:
@saiarcot895 , thanks for your comment, I think it should be good to remove deb.debian.org, updated.

Reply:
Also, what's the difference between debian-archive.trafficmanager.net and packages.trafficmanager.net? Are both needed/recommended?

Reply:
@saiarcot895 , not much different, but packages.trafficmanager.net is for reproducible build, which contains version frozen packages, debian-archive.trafficmanager.net may not contain them.